File: bmi.cpp
#include <iostream>

using namespace std;

int main()

{
   float weight;
   
   float height;
   
   float heightsquared;
   
   float weightoverheight;
   
   float bmi;
   
   cout << "BMI Calculator" << endl;
   
   cout << "What is your weight in pounds(lbs)?" << endl;
   
   cin >> weight;
   
   cout << "What is your height in inches(in.)?" << endl;
   
   cin >> height;
   
   heightsquared = height * height;
   
   weightoverheight = weight / heightsquared;
   
   bmi = weightoverheight * 703;
   
   cout << "Your BMI is...." << bmi;
   
   if (bmi<18.5)
   
      cout << ", your underweight!" << endl;
      
   if (bmi>30)
   
      cout << ", your obese!" << endl;
   
   if (bmi>18.5 && bmi<25)
   
      cout << ", your healthy!" << endl;
   
   if (bmi>=25 && bmi<=30)
      
      cout << ", your overweight!" << endl;


   return 0; 
}

File: bmi.py
def main():

    height = float(input("Enter your height in inches. "))
    
    weight = float(input("Enter your weight in pounds. "))


    heightsquared = (height * height)
    
    jeff = (weight / heightsquared)

    bmi = (jeff * 703.0704)


    print("Your bmi is....")
    print (bmi)
    
    if (bmi <= 18.5):
        print ("You are underweight.")

    if (18.5 < bmi < 25):
        print ("You are normal weight.")

    if (25 <= bmi < 30):
        print ("You are overweight.")
        
    if (30 <= bmi < 40):
        print ("You are obese.")
        
    if (bmi >= 40):
        print ("You are extremely obese.")

main()



